kotlin/vci-client/src/main/java/io/mosip/vciclient/proof/jwt/JWTProof.kt

```diff
@@ -15,9 +15,9 @@ import io.mosip.vciclient.proof.Proof
 import kotlin.math.floor
 
 private const val TOKEN_EXPIRATION_PERIOD_IN_MILLISECONDS = 18000
-private val logTag = Util.getLogTag(JWTProof::javaClass.name)
 
 class JWTProof : Proof {
+    private val logTag = Util.getLogTag(javaClass.simpleName)
```

Comment:
do we need private here or only val logTag will work?

Reply:
Since this logTag is not required by any other classes, added private access to it

Reply:
Initially I had more than one class in same file for which I had to keep the logTag outside class so all classes can have same logTag, later in refactoring the classes were separated out to different files, but I missed to move the logTag as a class field, So I moved it and keeping it as private since its not used by any class